Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessFieldedMetadata.java

```diff
@@ -676,16 +671,29 @@ public void preserve() throws InvalidMetadataValueException, NoSuchMetadataField
                 division.setLabel(null);
             }
             metadata.clear();
+            Set<String> specialFields = new HashSet<>(Set.of(METADATA_KEY_LABEL, METADATA_KEY_ORDERLABEL,
```

Comment:
Wouldn’t it be enough to assign directly? Further more, I would suggest to make this a constant

     private static final Set<String> SPECIAL_FIELDS = Set.of( ...